app/jobs/calendar_importer/calendar_importer_task.rb

```diff
@@ -28,6 +28,11 @@ def run
     all_event_uids = Set.new(calendar.events.upcoming.pluck(:uid))
     active_event_uids = Set.new
 
+    # this can be useful if you want to force import on your local machine
```

Comment:
can we make this a yarddoc-compatible message so it comes up in the docs when you search the function? this feels like important info basically and we should make it easier to find :)